libraries/hipcheck-sdk-macros/src/lib.rs

```diff
@@ -265,10 +265,6 @@ pub fn queries(_item: TokenStream) -> TokenStream {
 		};
 		agg.extend(out);
 	}
-	eprintln!(
-		"Auto-generating Plugin::queries() with {} detected queries",
-		q_lock.len()
-	);
```

Comment:
Since `queries! {}` relies on `#[query]` functions preceding its invocation, I had this print line as a sanity check for users to ensure the number of queries it sees is the right amount

Reply:
I am generally against printing during macro compilation by default. If we instead wanted to check an environment variable like `HC_DEBUG` and print if it's present, that would be fine.

Reply:
Using a debug env var works for me

Reply:
Done!